src/successModal/index.tsx

```diff
@@ -1,2 +1,4 @@
 export { SuccessModal, SuccessModalProps } from './SuccessModal'
+/** @Todo Only used in SPA test should be removed when SPA stop using this import */
+export { SuccessModalElements } from './SuccessModal.style'
```

Comment:
We check for a button.. with Jest, it wasn't RTL yet!?
Sorry, I haven't spot that missing export

Reply:
Yep I saw the test we'll remove it when migrated to RTL. It was a hard one to spot :)